core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -287,42 +287,21 @@ public static Wallet fromSeed(NetworkParameters params, DeterministicSeed seed,
     }
 
     /**
-     * Creates a wallet that tracks payments to and from the HD key hierarchy rooted by the given watching key. A
-     * watching key corresponds to account zero in the recommended BIP32 key hierarchy.
```

Comment:
`watching key corresponds to account zero in the recommended BIP32 key hierarchy.` was removed from the comments, because the path of the doesn't have to be 0H.